Packs/CrowdStrikeFalcon/Playbooks/playbook-CrowdStrike_Falcon_-_Retrieve_File.yml

```diff
@@ -230,8 +237,101 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 780,
-          "y": 700
+          "x": 1040,
+          "y": 1100
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: e5c3001b-7ebf-4e51-810f-c50c3ebd9274
+    type: condition
+    task:
+      id: e5c3001b-7ebf-4e51-810f-c50c3ebd9274
+      version: -1
+      name: Is CrowdStrike Falcon enabled?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'.
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "7"
+    scriptarguments:
+      brandname:
+        simple: CrowdstrikeFalcon
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: 0c9ac239-34cc-46eb-8662-7d573761dc7c
+    type: condition
+    task:
+      id: 0c9ac239-34cc-46eb-8662-7d573761dc7c
+      version: -1
+      name: Are all the inputs provided?
+      type: condition
+      iscommand: false
+      brand: ""
+      description: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "1"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
```

Comment:
let's add also the password here - cause if that won't be provided, the zip won't be extracted.

Reply:
But it provided by default in the inputs because it always "infected" in Crowdstrike